pkg/goDB/storage/gpfile/gpfile_test.go

```diff
@@ -249,7 +250,7 @@ func TestMetadataRoundTrip(t *testing.T) {
 
 	require.Nil(t, os.RemoveAll("/tmp/test_db"))
 
-	testDir := NewDir("/tmp/test_db", 1000, ModeWrite)
+	testDir := NewDirWriter("/tmp/test_db", 1000)
```

Comment:
Lots of `/tmp/test_db` mentions. Make a const?

Reply:
Ideally this would use `os.TempDir()` of course for portability. I'll see what I can quickly do, worst case I'll definitively use the proposed const at least.